x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/selectors.ts

```diff
@@ -72,6 +72,12 @@ export const patternsError = (state: Immutable<EndpointState>) => state.patterns
 const detailsPolicyAppliedResponse = (state: Immutable<EndpointState>) =>
   state.policyResponse && state.policyResponse.Endpoint.policy.applied;
 
+/**
+ * Returns the policy response timestamp from the endpoint after a user modifies a policy.
+ */
+export const policyResponseTimestamp = (state: Immutable<EndpointState>) =>
```

Comment:
can you add a response type to this function? Looks like its either the value of `@timestamp` or false/undefined? Probably should be something like: `Immutable<EndpointState>['policyResponse]['@timestamp']` ??